pkg/cmd/build/build.go

```diff
@@ -251,7 +251,7 @@ func OptsFromBuildInfo(manifestName, svcName string, b *build.Info, o *types.Bui
 
 	file := b.Dockerfile
 	if b.Context != "" && b.Dockerfile != "" {
-		file = extractFromContextAndDockerfile(b.Context, b.Dockerfile, svcName)
+		file = extractFromContextAndDockerfile(b.Context, b.Dockerfile, svcName, os.Getwd)
```

Comment:
why are we using `os.Getwd` directly, instead of using afero? 🤔

Reply:
afero doesn't have `os.Getwd` and I had to mock the call to the system so I added as param so I could test the function